src/printer.js

```diff
@@ -878,11 +878,13 @@ function genericPrintNoParens(path, options, print, args) {
     case "ThisExpression":
       return "this";
     case "Super":
-      return "super"; // Babel 6 Literal split
+      return "super";
+    // Babel 6 Literal split
```

Comment:
My guess is that I have restored the original comment position after Prettier had moved it to the end of the previous line some time.